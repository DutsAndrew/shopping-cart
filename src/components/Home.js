import React, { useState, useEffect } from "react";
import '../style/Home.css';

const Home = props => {
  return (
    <div>
      <p>Home</p>
    </div>
  );
};

export default Home;