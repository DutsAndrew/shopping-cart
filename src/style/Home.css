#home-container {
  display: flex;
  flex-flow: column wrap;
  justify-content: flex-start;
  align-items: center;
  flex: auto;
  width: 100vw;
  height: 85vh;
  font-size: 1.25em;
}