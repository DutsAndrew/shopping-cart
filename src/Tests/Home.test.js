import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Home from '../components/Home';

describe('home component', () => {
  it('renders home page with paragraphs', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  })
});