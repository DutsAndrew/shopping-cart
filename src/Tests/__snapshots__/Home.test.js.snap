// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component renders home page with paragraphs 1`] = `
<div>
  <div
    id="home-container"
  >
    <p
      id="welcome message"
    >
      Welcome to Steam Lite
    </p>
    <p
      id="store-message"
    >
      By clicking on the "Shop" page at the top you can browse games in the Steam catalog
    </p>
    <p
      id="store-message-pt2"
    >
      Happy Gaming!
    </p>
    <p
      id="disclaimer-message"
    >
      This site is note an actual Steam site. This is just a project to practice my coding.
    </p>
  </div>
</div>
`;
